app/assets/javascripts/oxalis/controller/annotations/volumetracing_controller.js

```diff
@@ -54,62 +50,13 @@ class VolumeTracingController {
       },
     });
 
-    // no merging for now
-    $("#btn-merge").hide();
-
     this.mergeMode = this.MERGE_MODE_NORMAL;
-    const isMergeVisible = () => $("#merge").css("visibility") === "visible";
```

Comment:
No merger mode anymore?

Reply:
At one point we had the basics for merger mode for volume tracings but it was never enabled. The code laid dormant and we recently removed it. This was left over from the GUI part of that.